lumispy/signals/common_luminescence.py

```diff
@@ -38,42 +40,26 @@
 class CommonLumi:
     """**General luminescence signal class (dimensionless)**"""
 
-    def crop_edges(self, crop_px):
-        """Crop the amount of pixels from the four edges of the scanning
-        region, from out the edges inwards.
+    def crop_edges(self, crop_range=None, crop_px=None, rebin_nav=False, **kwargs):
+        """Crop edges along the navigation axes of the signal object.
+
+        %s
 
         Parameters
         ----------
-        crop_px : int
-            Amount of pixels to be cropped on each side individually.
-
-        Returns
-        -------
-        signal_cropped : CommonLuminescence
-            A smaller cropped CL signal object. If inplace is True, the original
-            object is modified and no LumiSpectrum is returned.
-        """
 
-        width = self.axes_manager.shape[0]
-        height = self.axes_manager.shape[1]
-
-        if crop_px * 2 > width or crop_px * 2 > height:
-            raise ValueError(
-                "The pixels to be cropped cannot be larger than half the width or the length!"
+        %s
```

Comment:
I did some corrections to the formatting of docstrings, but there is still a indentation-related error that leads to wrong display of the parameters in the docstring:
https://lumispy--237.org.readthedocs.build/en/237/api/lumispy.signals.common_luminescence.html#lumispy.signals.common_luminescence.CommonLumi.crop_edges